tests/api/v3-endpoint-naming.test.ts

```
@@ -0,0 +1,528 @@
+};
+
+describe('V3.0 Compliant API Endpoint Naming', () => {
+  let app: Hono;
+
+  beforeEach(() => {
+    app = new Hono();
+    
+    // Create v3 router with mocked dependencies
+    const v3Router = createVersionedRouter(
+      BadgeVersion.V3,
+      mockIssuerController as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)